integration_tests/tests/flowmachine_server_tests/conftest.py

```diff
@@ -11,17 +11,30 @@
 
 
 @pytest.fixture(scope="session")
-def zmq_url():
+def zmq_host():
     """
-    Return the URL where to connect to zeromq when running the tests.
+    Return the host on which zmq is running. This is either the value of
+    the environment variable FLOWMACHINE_ZMQ_HOST or else 'localhost'.
+    """
+    return os.getenv("FLOWMACHINE_ZMQ_HOST", "localhost")
+
 
-    By default this is "tcp://localhost:5555" but the tests can be run
-    against a flowmachine at a different URL by setting the environment
-    variable FLOWMACHINE_ZMQ_URL.
+@pytest.fixture(scope="session")
+def zmq_port():
+    """
+    Return the port on which zmq is running. This is either the value of
+    the environment variable FLOWMACHINE_ZMQ_PORT or else 5555.
+    """
+    return os.getenv("FLOWMACHINE_ZMQ_PORT", "5555")
+
+
+@pytest.fixture(scope="session")
+def zmq_url(zmq_host, zmq_port):
+    """
+    Return the URL where to connect to zeromq when running the tests.
+    This is currently hard-coded to "tcp://localhost:5555".
```

Comment:
Is this hard-coded? Looks like setting `FLOWMACHINE_ZMQ_HOST` and `FLOWMACHINE_ZMQ_PORT` should override this.

Reply:
Indeed 👍